src/Symfony/Component/Semaphore/Tests/Store/AbstractStoreTestCase.php

```diff
@@ -204,7 +204,6 @@ public function testPutOffExpirationWhenSaveHasNotBeenCalled()
         $key1 = new Key(__METHOD__, 4, 2);
 
         $this->expectException(SemaphoreExpiredException::class);
-        $this->expectExceptionMessage('The semaphore "Symfony\Component\Semaphore\Tests\Store\AbstractStoreTestCase::testPutOffExpirationWhenSaveHasNotBeenCalled" has expired: the script returns a positive number.');
```

Comment:
This exception message looks adapter specific and so I did remove this check. As "the script returns a positive number." is something redis specific.